developer/src/kmc/test/fixtures/sections/meta/invalid-version-1.0.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!DOCTYPE keyboard SYSTEM "../../../../../../../resources/standards-data/ldml-keyboards/techpreview/ldmlKeyboard.dtd">
+<keyboard locale="mt" conformsTo="techpreview">
+  <version number="1.0" />
```

Comment:
invalid because it's not 1.0.0 as required by semver

Reply:
```suggestion
  <version number="1.0" /> <!-- invalid because it's not "1.0.0" as required by semver -->
```